src/tokenizer.ts

```diff
@@ -1362,6 +1362,11 @@ export class Tokenizer extends DiagnosticEmitter {
               : DiagnosticCode.Multiple_consecutive_numeric_separators_are_not_permitted,
             this.range(pos)
           );
+        } else if (pos - 1 == start && text.charCodeAt(pos - 1) == CharCode._0) {
```

Comment:
Is `00_something` valid?

Reply:
no. It handled before as `ERROR 1121: "Octal literals are not allowed in strict mode.`

Reply:
I see. Can we add a test case just to be sure?

Reply:
sure!